sw/device/silicon_creator/rom/rom.ld

```diff
@@ -105,6 +105,20 @@ SECTIONS {
     . = ALIGN(4);
   } > rom
 
+  /**
+   * ROM ISRs text section, containing the exception handler entry/exit logic
+   * used for handling load access faults due to flash ECC errors during boot.
+   *
+   * This is placed next to the ".shutdown" section.
+   */
+  .rom_isrs : ALIGN(4) {
```

Comment:
This LGTM, but open to discuss in case we want to collapse this into another region.

Reply:
Ok.  Lets leave it for now.